src/web/src/components/CourseListing.vue

```diff
@@ -11,15 +11,14 @@
         <button
           v-if="actions.add || actions.remove"
           class="btn"
-          @click="toggleCourse(course)"
+          @click="actions.remove ? toggleCourse(course):null"
```

Comment:
this should be breaking the course list 
it should be sufficient to change `@click` to `@click.stop` to prevent the click event from propagating to the parent div

Reply:
This click event needs to happen if it's a sections listing

Reply:
Why would it be breaking the course list? If action.remove, the button toggles the course, else it does nothing

Reply:
ok my bad, it doesn't break because both `SelectedCourses` and `CourseList` provide both `actions.add` and `actions.remove`. In this case, if the `CourseListing` was only provided `actions.add`, then the add wouldn't work.

Reply:
Hmm? Course list provides .add & Selected courses provides .remove. That's what I am checking for here

Reply:
Nevermind I'm wrong you're correct. but i'd much prefer if you used `@click.stop` instead